config/routes.rb

```diff
@@ -137,6 +136,18 @@
 
   match "/oembed", to: 'master_files#oembed', via: [:get]
 
+  def can_see?(thing, scope=nil)
```

Comment:
I think this is too narrow a scope.  I think we'd want to be able to declare new abilities and be able to use them here like:

def route_can?(action, thing, scope=nil)
constraints(route_can?(:read, :about_page))